modules/logging-audit-log-filtering.adoc

```diff
@@ -12,6 +12,12 @@ OpenShift API servers generate audit events for each API call, detailing the req
 * `Request`: Audit metadata and the request body are included, the response body is removed.
 * `RequestResponse`: All data is included: metadata, request body and response body. The response body can be very large. For example, `oc get pods -A` generates a response body containing the YAML description of every pod in the cluster.
 
+
+[NOTE]
+====
+Using this feature with Fluentd is not supported.
```

Comment:
Consider counter wording to this since we are trying to remove all references to fluentd.  consider saying this Feature is only supported with Vector collector implementations

Reply:
sure @jcantrill will make those changes!